src/components/tabs/FunctionsTab.vue

```diff
@@ -1,18 +1,47 @@
 <template>
-  <div>
-    Functions stuff...
-    <div>
-      <UIButton text="Upload File" @click="uploadFile"/>
-      <input
-        type="file"
-        id="upload-python-file"
-        style="display: none"
-        @change="load"
-      >
+  <div class="d-flex h-100">
+    <div id="left" class="panel">
+      <div class="d-flex">
+        <div>
+          <UIButton text="Upload File" @click="uploadFile"/>
+          <input
+            type="file"
+            id="upload-python-file"
+            style="display: none"
+            @change="load"
+          >
+        </div>
+        <div>
+          <UIButton text="Create File" @click="createFile"/>
+        </div>
+      </div>
+      <div class="button-list">
+        <FileFuncButton v-for="(file, index) of files"
+                        :header="`${file.filename} (${file.functions.length})`"
+                        :key="index"
+                        :selected="selectedFile === index"
+                        @click="selectFile(index)">
+          {{  getFunctions(index).length > 0 ? getFunctions(index)[0].signature() : '(empty)' }}
```

Comment:
Can we maybe have this in a monospaced font maybe? I think this will look nice with the code!

Reply:
Yeah that's a good idea, I'll add that!